src/Vlc.DotNet.Core/VlcMediaPlayer/VlcMediaPlayer.cs

```diff
@@ -290,6 +290,11 @@ public void TakeSnapshot(FileInfo file, uint width, uint height)
             Manager.TakeSnapshot(myMediaPlayerInstance, file, width, height);
         }
 
+        public void FullScreen()
+        {
+            Video.FullScreen = true; //added property.
```

Comment:
I'm not sure we must keep this. Either we need to implement this as a property, and forward the boolean value, or remove this entirely and let the user call Video.Fullscreen

Reply:
removed the method. User can use the Video.FullScreen property directly.